python/lsst/jointcal/utils.py

```diff
@@ -124,9 +130,10 @@ def rms_total(data):
         self.old_dist_total = MatchDict(*(tuple(map(rms_total, self.old_dist))*u.radian).to(u.arcsecond))
         self.new_dist_total = MatchDict(*(tuple(map(rms_total, self.new_dist))*u.radian).to(u.arcsecond))
 
-        return self.new_dist_total.relative, self.new_dist_total.absolute
+        Rms_result = collections.namedtuple("rms_result", ["dist_relative", "dist_absolute", "pa1"])
```

Comment:
A namedtuple is like a "mini-class", so should be capitalized (see the [example in the docs](https://docs.python.org/2/library/collections.html#collections.namedtuple)). That said, I should have called it "Rms_result" in the namedtuple constructor, which I have fixed.